lesson_13/bank/bank_app/src/main/java/com/codedifferently/lesson13/bank/AuditLog.java

```diff
@@ -0,0 +1,39 @@
+package com.codedifferently.lesson13.bank.audit;
+import java.util.Set;
+import java.util.UUID;
+
+import com.codedifferently.lesson13.bank.BankAccountBase;
+import com.codedifferently.lesson13.bank.BankAtm;
+import com.codedifferently.lesson13.bank.CheckingAccount;
+import com.codedifferently.lesson13.bank.Customer;
+
+
+public class AuditLog extends BankAtm {
```

Comment:
Not sure what you were attempting here, but this doesn't look like it was going in the right direction. This didn''t need to extend `BankAtm`, would've just been fine as an object with methods for logging debits and credits (just logging that a debit or credit happened works fine).